packages/database/src/utils/fhir/Encounter/getValues.ts

```diff
@@ -116,10 +116,27 @@ function subjectRef(encounter: Encounter, models: Models) {
   });
 }
 
-const { BED, WARD } = FHIR_LOCATION_PHYSICAL_TYPE_CODE;
+const { BED, WARD, JURISDICTION } = FHIR_LOCATION_PHYSICAL_TYPE_CODE;
 
-function locationRef(encounter: Encounter) {
+async function locationRef(encounter: Encounter, models: Models) {
+  const department = await models.Department.findOne({ where: { id: encounter.departmentId } });
```

Comment:
@reviewer I feel there's meant to be a way we can auto-link the department to the encounter so that you can just do `encounter.department.name` but I couldn't work out how to do it :(

Reply:
oh yeah that is a shame. I thought that `getFullReferenceAssociations` did that for us but all good